internal/template-bundle/bundle.go

```diff
@@ -73,9 +73,10 @@ func readTemplates(filename string) ([]templatev1.Template, error) {
 		if !ok {
 			return nil, err
 		}
-		// DISCLAIMER: This is a temporary solution for delivering templates related to the host architecture.
-		// Once the common templates are released based on architecture, this change will no longer be necessary.
-		// Instead, a modification will be required in setup.go to specify the bundle to read from.
+		// DISCLAIMER: This change is necessary for templates related to unsupported
+		// architectures. By implementing this, we ensure that we always filter by
+		// architecture. Additionally, the releases of a new architecture template
+		// can be managed separately from the generic template.
```

Comment:
I'm unsure what you want to say here. 
The comment starts with `This change is necessary...`, but nothing is changing in this piece of code. It only filters templates by architecture.